skin/frontend/base/default/js/catalin_seo/handler.js

```diff
@@ -142,6 +142,15 @@ var CatalinSeoHandler = {
         document.observe("dom:loaded", function () {
             self.ajaxListener();
 
+            $j('a.show_more_filters').on('click', function (e) {
+                $j(e.target).parent().parent().find('.filter_hide').toggle();
+                if($j(e.target).text() == $j(e.target).data('text-more')) {
+                    $j(e.target).text($j(e.target).data('text-less'));
+                } else {
+                    $j(e.target).text($j(e.target).data('text-more'));
+                }
+            });
+
```

Comment:
I think that this code should be moved to a separate method, like **showMoreListener** and called in **bindListeners** like now but also somewhere after line 170 and in method **handleEvent** after line 55.